example/docker-compose/local/readme.md

```diff
@@ -10,7 +10,6 @@ permissions set in order to start correctly.
 
 ```console
 mkdir tempo-data/
-sudo chown 10001:10001 tempo-data/
```

Comment:
Why is this removed? Is it covered by `init` in the compose?

Reply:
If you do that the container won't start due to permissions errors. Just creating the folder and allowing the init to chown the folder is enough